resources/views/livewire/dropzone.blade.php

```diff
@@ -2,7 +2,8 @@
     x-cloak
     x-data="dropzone({
         _this: @this,
-        multiple: @js($multiple)
+        multiple: @js($multiple),
+        uuid: '{{ $uuid }}'
```

Comment:
Instead of `uuid: '{{ $uuid }}'`, you can use `uuid: @js($uuid)`.

Reply:
I ran into some weird issues like that when I didn't specify a key. By specifying ":key" essentially every issue I encountered went away. The thing to note is the key must be unique. I did this by just appending an extra character if you have multiple components being keyed in a loop like I do.  Let me know if this fixes things.
 
![image](https://github.com/dasundev/livewire-dropzone/assets/11857985/205747c0-70e4-4802-b0d3-33199227030b)

Reply:
I tried with `:key`, it didn't solve the mentioned problems from my end.

Reply:
Odd. I am running into none of those issues. The dropzone component has been working pretty flawlessly after making those fixes. 

I can't show too much of the code I am working on but when I have the time, I will create a small example laravel project to show I am implementing this.

Reply:
Everything looks good. The issue was on my end. I'll make the rest of the changes and merge the PR.